cmd/apps/gitea_app.go

```diff
@@ -0,0 +1,131 @@
+		Long:         `Install gitea`,
+		Example:      `  arkade install gitea`,
+		SilenceUsage: true,
+	}
+
+	gitea.Flags().Bool("update-repo", true, "Update the helm repo")
+	gitea.Flags().String("namespace", "default", "Kubernetes namespace for the application")
+	gitea.Flags().Bool("persistence", false, "Enable persistence")
+	gitea.Flags().String("admin-user", "gitea_admin", "Username of admin user")
+	gitea.Flags().String("admin-password", "", "Overide the default random admin-password if this is set")
+	gitea.Flags().StringArray("set", []string{},
+		"Use custom flags or override existing flags \n(example --set persistence.enabled=true)")
+
+	gitea.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath := config.GetDefaultKubeconfig()
+
+		if command.Flags().Changed("kubeconfig") {
+			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
+		}
+		updateRepo, _ := gitea.Flags().GetBool("update-repo")
+
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+
+		arch := k8s.GetNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		if arch != IntelArch {
+			log.Printf("If installing on ARM platform you'll need to use an external database")
+		}
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		clientArch, clientOS := env.GetClientArch()
+
+		fmt.Printf("Client: %s, %s\n", clientArch, clientOS)
+		log.Printf("User dir established as: %s\n", userPath)
+
+		os.Setenv("HELM_HOME", path.Join(userPath, ".helm"))
+
+		ns, _ := gitea.Flags().GetString("namespace")
+
+		persistence, _ := gitea.Flags().GetBool("persistence")
+		overrides := map[string]string{}
+
+		overrides["persistence.enabled"] = strings.ToLower(strconv.FormatBool(persistence))
+
+		pass, _ := command.Flags().GetString("basic-auth-password")
+
+		if len(pass) == 0 {
+			var err error
+			pass, err = password.Generate(25, 10, 0, false, true)
+			if err != nil {
+				return err
+			}
+		}
+
+		overrides["gitea.admin.password"] = pass
+
+		adminUsername, _ := command.Flags().GetString("admin-user")
+		overrides["gitea.admin.username"] = adminUsername
+
+		customFlags, err := gitea.Flags().GetStringArray("set")
+		if err != nil {
+			return fmt.Errorf("error with --set usage: %s", err)
+		}
+
+		if err := mergeFlags(overrides, customFlags); err != nil {
+			return err
+		}
+
+		giteaAppOptions := types.DefaultInstallOptions().
+			WithNamespace(ns).
+			WithHelmPath(path.Join(userPath, ".helm")).
+			WithHelmRepo("gitea-charts/gitea").
+			WithHelmURL("https://dl.gitea.io/charts").
+			WithOverrides(overrides).
+			WithHelmUpdateRepo(updateRepo)
+
+		_, err = helm.TryDownloadHelm(userPath, clientArch, clientOS)
+		if err != nil {
+			return err
+		}
+
+		_, err = apps.MakeInstallChart(giteaAppOptions)
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(giteaInstallMsg)
+		return nil
+	}
+
+	return gitea
+}
+
+var giteaInstallMsg = `=======================================================================
```

Comment:
Please can you add an info message as per the openfaas app?

Reply:
added